src/components/confirmation-dialogs.js

```diff
@@ -283,6 +283,7 @@ export function reloginAfterSessionExpired(modal, t, onOk, onCancel) {
   const formRef = createRef();
 
   let dialog = null;
+  let isFrozen = false;
```

Comment:
This variable should maybe have a clearer name since the module is not a "login" dedicated context. Maybe `isLoginFrozen` or `isLoggingInFrozen` to correlate with `isLogginIn`?

Reply:
Agree. Changed it to `isLoginFrozen`.